### PR TITLE
crux_core 0.8.0-rc.1

### PR DESCRIPTION
**Pull request by StuartHarris**
`crux_core` v0.8.0-rc.1

### CODE REVIEW

File: crux_http/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "crux_http"
 description = "HTTP capability for use with crux_core"
-version = "0.9.1"
+version = "0.9.2"
```
Comment (obmarg) [inline crux_http/Cargo.toml:4]: The changelog says 0.9.2-rc.0 but this is just 0.9.2, I assume one of them is wrong

File: crux_kv/Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "crux_kv"
 description = "Key-Value capability for use with crux_core"
-version = "0.4.0"
+version = "0.4.1-rc.1"
```
Comment (obmarg) [inline crux_kv/Cargo.toml:4]: Changelog for this one says 0.4.1-rc.0

File: crux_macros/Cargo.toml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "crux_macros"
 description = "Macros for use with crux_core"
-version = "0.3.9"
+version = "0.3.10-rc.1"
```
Comment (obmarg) [inline crux_macros/Cargo.toml:4]: Changelog for this one says 0.3.10-rc.0

File: crux_http/Cargo.toml (hunk 1 above)
Comment (StuartHarris) [reply crux_http/Cargo.toml]: Thanks! my bad

**Review by adwhit — APPROVED** [review]

**Review by charypar — APPROVED** [review]
I _think_ that looks right 😅

**Pull request merged**